tools/config_manager.py

```diff
@@ -251,3 +254,15 @@ def _update_activation_config(to_update_data, current_config, config_file_path,
     @staticmethod
     def has_invalid_default_config_value(*config_values):
         return any(value in DEFAULT_CONFIG_VALUES for value in config_values)
+
+    @staticmethod
+    def get_market_pair(config, currency):
+        reference_market = config[CONFIG_TRADING][CONFIG_TRADER_REFERENCE_MARKET]
```

Comment:
I think we need something to 'get or default' the reference market from the config.
I think the order manager can be the good place for this.

Reply:
Good idea for the get or default but doing it in config manager is better in my opinion because it's data only from the config file and config/__init__.py which has a default reference market (that I forgot about)